modelbaker/dbconnector/mssql_connector.py

```diff
@@ -1033,3 +1033,41 @@ def get_ili2db_settings(self):
             )
             result = self._get_dict_result(cur)
         return result
+
+    def get_ili2db_sequence_value(self):
+        if self.schema:
+            cur = self.conn.cursor()
+            cur.execute(
+                """
+                    SELECT NEXT VALUE FOR {schema}.{sequence};
+                """.format(
+                    schema=self.schema, sequence="t_ili2db_seq"
+                )
+            )
+            content = cur.fetchone()
+            if content:
+                return content[0]
+        return None
+
+    def set_ili2db_sequence_value(self, value):
+        if self.schema:
+            cur = self.conn.cursor()
+            try:
+                cur.execute(
+                    """
+                    ALTER SEQUENCE {schema}.{sequence} RESTART WITH {value};
+                    """.format(
+                        schema=self.schema, sequence="t_ili2db_seq", value=value
+                    )
+                )
+                self.conn.commit()
+                return True, self.tr(
+                    'Successfully reset sequence value to "{}".'
+                ).format(value)
+            except pyodbc.errors.Error as e:
+                error_message = " ".join(e.args)
+                return False, self.tr("Could not reset sequence: {}").format(
```

Comment:
Would not be better to throw an exception instead of returning false? I'm thinking more general that only this case. Or is it a design décision for the library?

Reply:
Oopsi. Haven't seen this before merging. Good input, but I wanted it to make it identical to other functions. Some (e.g.) create_baskets are called for multiple baskets, where in  the frontend I 'collect' all the feedbacks and return it in a list. Kind of "5 of 10 baskets are not created" or similar.